### ISSUE TITLE
undefined method `register' for GoogleTagManager:Class

### ISSUE THREAD

**Issue by jevy** [rjocoleman/middleman-google-tag-manager#1]
Hiya!  I know adding GTM is simple but thought might as well use your gem.  I'm getting this:

```
➜  git:(master) ✗ bundle exec middleman serve
RubyDep: WARNING: your Ruby is outdated/buggy. Please upgrade. (To disable warnings, set RUBY_DEP_GEM_SILENCE_WARNINGS=1)
bundler: failed to load command: middleman (/Users/jevin.maltais/.rbenv/versions/2.2.4/bin/middleman)
Bundler::GemRequireError: There was an error while trying to load the gem 'middleman-google-tag-manager'.
Gem Load Error is: undefined method `register' for GoogleTagManager:Class
Backtrace for gem load error is:
/Users/jevin.maltais/.rbenv/versions/2.2.4/lib/ruby/gems/2.2.0/gems/middleman-google-tag-manager-0.1.0/lib/middleman-google-tag-manager.rb:53:in `<top (required)>'
...
```

**Comment by rjocoleman** [comment]
Weird! I just wrote this a few days ago.. What version of middleman?

**Comment by rjocoleman** [comment]
closed via #2
